src/main/scala/com/nec/spark/planning/OneStageEvaluationPlan.scala

```diff
@@ -78,6 +78,16 @@ final case class OneStageEvaluationPlan(
 
   override def output: Seq[Attribute] = outputExpressions.map(_.toAttribute)
 
+  private val isProject = veFunction.functionName.startsWith("project")
```

Comment:
Can remove these now

Reply:
Yeah, somehow this got messed up during rebas.e